requirements.txt

```diff
@@ -1,11 +1,15 @@
 annotated-types==0.5.0
 anyio==3.7.1
 fastapi==0.101.1
```

Comment:
This is quite a lot of things, can we remove stuff we aren't using (like fast api)

Reply:
Removed all necessary packages.